CHANGES.md

```diff
@@ -11,6 +11,8 @@
 <!-- Changes that affect Black's stable style -->
 
 - Fix comments getting removed from inside parenthesized strings (#3909)
+- Parenthesize singleton tuples in `yield` expressions as they already are in `return`
```

Comment:
This needs to go in the preview style, not the stable style. (That's also why CI is failing.)